### PR TITLE
feat: add --releaseDate to git node release

### PR DESCRIPTION
**Pull request by RafaelGSS**
This will enable us to automate release proposal
creation: https://github.com/nodejs/security-wg/issues/860

### CODE REVIEW

File: lib/prepare_release.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export default class ReleasePreparation extends Session {
     this.isLTS = false;
     this.isLTSTransition = argv.startLTS;
     this.runBranchDiff = !argv.skipBranchDiff;
+    this.defaultReleaseDate = argv.releaseDate ?? new Date().toISOString().split('T')[0];
```
Comment (aduh95) [inline lib/prepare_release.js:28]:
```suggestion
    this.defaultReleaseDate = argv.releaseDate ?? new Date().toISOString().slice(0, 10);
```

File: docs/git-node.md [hunk 2]
```diff
@@ -336,7 +338,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
+Update or patch the V8 engine.
```
Comment (aduh95) [inline docs/git-node.md:341]: Unrelated to this PR, and likely undesirable (the two spaces forces the markdown rendering tools to include a line break)
```suggestion
Update or patch the V8 engine.  
```

File: docs/git-node.md [hunk 3]
```diff
@@ -373,7 +375,7 @@ Options:
 ### `git node v8 minor`
 
 Compare current V8 version with latest upstream of the same major. Applies a
-patch if necessary.  
+patch if necessary.
```
Comment (aduh95) [inline docs/git-node.md:378]: ditto
```suggestion
patch if necessary.  
```

**Comment by RafaelGSS** [comment]
I think `npm run lint-fix` did that btw.

**Review by aduh95 — APPROVED** [review]

**Pull request merged**